sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2237,10 +2237,10 @@ public virtual async Task<AnalyzeHealthcareEntitiesOperation> StartAnalyzeHealth
         public virtual Response<ClassificationCategoryCollection> DynamicClassify(
             string document,
             IEnumerable<string> categories,
+            ClassificationType? classificationType = default,
             string language = default,
```

Comment:
This hasn't GA'd yet, right? If it has, don't change the order and use an overload. If it hasn't (which I think is the case), I would still consider putting the language third for consistency. Or, is it consistent that it is last (before a cancellation token)? I don't recall.

Reply:
It hasn't GA'd so reordering is okay. Regarding consistency, the pattern that I think I'm seeing appears to determine the position of the `language` property by counting from the last parameter to the first with two possible options:
1. `cancellationToken` and `language` (does not receive an `options` parameter)
2. `cancellationToken`, `options`, and `language`.

For example, the `StartSingleLabelClassify` method looks like this: 
```csharp
public virtual ClassifyDocumentOperation StartSingleLabelClassify(
    IEnumerable<string> documents,
    string projectName,
    string deploymentName,
    string language = default,
    SingleLabelClassifyOptions options = default,
    CancellationToken cancellationToken = default)
```

Reply:
Right, so seems `options` should go last. In fact, I thought we had a guideline about that: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-parameters

Turns out, though, we don't document position. But, IIRC, conversations with @KrzysztofCwalina and/or @tg-msft in the past said it should be next to last, where last is the `cancellationToken`.

Reply:
Yes, I believe this is correct! I have `options` (where applicable) next to last, and last is always `cancellationToken`:


```csharp
public virtual Azure.Response<Azure.AI.TextAnalytics.ClassificationCategoryCollection>
    DynamicClassify(
        string document,
        System.Collections.Generic.IEnumerable<string> categories,
        Azure.AI.TextAnalytics.ClassificationType? classificationType = default,
        string language = default,
        System.Threading.CancellationToken cancellationToken = default)
```
```csharp
public virtual System.Threading.Tasks.Task<Azure.Response<Azure.AI.TextAnalytics.ClassificationCategoryCollection>> 
    DynamicClassifyAsync(
        string document,
        System.Collections.Generic.IEnumerable<string> categories,
        Azure.AI.TextAnalytics.ClassificationType? classificationType = default, 
        string language = default,
        System.Threading.CancellationToken cancellationToken = default)
```
```csharp
public virtual Azure.Response<Azure.AI.TextAnalytics.DynamicClassifyDocumentResultCollection> 
    DynamicClassifyBatch(
        System.Collections.Generic.IEnumerable<Azure.AI.TextAnalytics.TextDocumentInput> documents,
        System.Collections.Generic.IEnumerable<string> categories,
        Azure.AI.TextAnalytics.ClassificationType? classificationType = default, 
        Azure.AI.TextAnalytics.TextAnalyticsRequestOptions options = default,
        System.Threading.CancellationToken cancellationToken = default)
```
```csharp
public virtual Azure.Response<Azure.AI.TextAnalytics.DynamicClassifyDocumentResultCollection> 
    DynamicClassifyBatch(
        System.Collections.Generic.IEnumerable<string> documents,
        System.Collections.Generic.IEnumerable<string> categories,
        Azure.AI.TextAnalytics.ClassificationType? classificationType = default, 
        string language = default,
        Azure.AI.TextAnalytics.TextAnalyticsRequestOptions options = default,
        System.Threading.CancellationToken cancellationToken = default) 
```
```csharp
public virtual System.Threading.Tasks.Task<Azure.Response<Azure.AI.TextAnalytics.DynamicClassifyDocumentResultCollection>>
    DynamicClassifyBatchAsync(
        System.Collections.Generic.IEnumerable<Azure.AI.TextAnalytics.TextDocumentInput> documents, 
        System.Collections.Generic.IEnumerable<string> categories,
        Azure.AI.TextAnalytics.ClassificationType? classificationType = default,
        Azure.AI.TextAnalytics.TextAnalyticsRequestOptions options = default,
        System.Threading.CancellationToken cancellationToken = default)
```
```csharp
public virtual System.Threading.Tasks.Task<Azure.Response<Azure.AI.TextAnalytics.DynamicClassifyDocumentResultCollection>> 
    DynamicClassifyBatchAsync(
        System.Collections.Generic.IEnumerable<string> documents,
        System.Collections.Generic.IEnumerable<string> categories,
        Azure.AI.TextAnalytics.ClassificationType? classificationType = default,
        string language = default,
        Azure.AI.TextAnalytics.TextAnalyticsRequestOptions options = default,
        System.Threading.CancellationToken cancellationToken = default)
```